scipy/_lib/doccer.py

```diff
@@ -80,14 +94,14 @@ def inherit_docstring_from(cls):
 
     Parameters
     ----------
-    cls : Python class or instance
+    cls : object
```

Comment:
Technically a `type` is also and `object`, but it maybe it could help to explicitly include `type` anyway

Reply:
I'm not exactly sure what to put here then, something like `object | type` or just `type`?

Reply:
yea something like that